### DISCUSSION TITLE
Implement of the tuya's compatible door locks ?

### DISCUSSION THREAD

**Discussion by SquidOx** [milo526/homebridge-tuya-web#279] [Q&A]
Hi,
I would like to know if you plan to implement the tuya compatible door locks in the plugin.

Thank you in advance for all responses

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.
